src/vs/workbench/contrib/welcome/page/browser/welcomePage.contribution.ts

```diff
@@ -25,7 +25,7 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration)
 				'scope': ConfigurationScope.APPLICATION, // Make sure repositories cannot trigger opening a README for tracking.
 				'type': 'string',
 				'enum': [
-					...['none', 'welcomePage', 'welcomePageWithTour', 'readme', 'newUntitledFile', 'welcomePageInEmptyWorkbench'],
```

Comment:
please add a {{SQL CARBON EDIT}} comment since this is in vs folder, also in PR description, you mentioned you removed the getting started option, but I don't see a code change relate to that.

Reply:
It helps if I push all the changes I have committed locally 😄

Reply:
Although guess we were missing the edit tag for this line too from the beginning so I'll add that as well